### ISSUE TITLE
openssl_random_pseudo_bytes documentation update

### ISSUE THREAD

**Issue by CJDennis** [php/doc-en#1523]
From manual page: https://php.net/function.openssl-random-pseudo-bytes

Return Values[ ¶](https://www.php.net/manual/en/function.openssl-random-pseudo-bytes.php#refsect1-function.openssl-random-pseudo-bytes-returnvalues)
Returns the generated string of bytes on success, or false on failure.

---

As of PHP 7.4, this is no longer true. Instead, an error is raised.

Return Values[ ¶](https://www.php.net/manual/en/function.openssl-random-pseudo-bytes.php#refsect1-function.openssl-random-pseudo-bytes-returnvalues)
Returns the generated string of bytes on success, or throws an Error on failure (e.g. if the number of bytes requested is less than 1).

**Comment by cmb69** [comment]
Indeed, `false` was only returned prior to PHP 7.4.0, while PHP 7.4 threw an `Exception`. As of PHP 8.0.0, either an `Exception` or a `ValueError` are thrown.
